Makefile

```diff
@@ -57,13 +57,9 @@ virtualenv: ## Creates virtualenv
 	@mkdir -p $(VIRTUALENV)
 	virtualenv --python $(PYTHON) $(VIRTUALENV)
 	$(PIP) install --upgrade pip
-# As to why no identation see https://stackoverflow.com/questions/4483313/make-error-for-ifeq-syntax-error-near-unexpected-token
-ifeq ($(UNAME), Linux)
-	$(PIP) install libpecos==0.1.0
-endif
 	$(PIP) install pytest pytest-cov tox
 	$(PIP) install -r requirements.txt
-	$(PIP) install --no-deps -e .
+	$(PIP) install -e .
```

Comment:
keep the `--no-deps` and if you need extra deps you need to add in `unpinned` and run `make update-requirements`